pkg/extensions/sync/signatures.go

```diff
@@ -272,7 +318,15 @@ func (sig *signaturesCopier) syncNotarySignature(localRepo, remoteRepo, digestSt
 		// read manifest
 		var artifactManifest oras.Manifest
 
-		err = json.Unmarshal(resp.Body(), &artifactManifest)
+		body, err := ioutil.ReadAll(resp.Body)
```

Comment:
you repeat this code reading the body, also the code which creates the request.

can you make something similar with cli makeHTTPGetRequest() where you give headers, url, basicAuth, etc

also you didn't migrate basicAuth, with resty it was set on client, now I think you need to set it on all http requests.

Reply:
simple check is when you do `make binary` it lists all deps it compiles against, so make sure resty doesn't show up there